fury/primitive.py

```diff
@@ -626,6 +626,54 @@ def prim_triangularprism():
     return vertices, triangles
 
 
+def prim_pentagonalprism():
+    """Return vertices and triangles for a pentagonal prism.
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our prism
+    triangles: ndarray
+        triangles that compose our prism
+
+    """
+    # Local variable to represent the square root of five
+    five = math.sqrt(5)
+    onec = (five - 1) / 4.0
+    twoc = (five + 1) / 4.0
+    sone = (math.sqrt(10 + (2 * five))) / 4.0
+    stwo = (math.sqrt(10 - (2 * five))) / 4.0
+
+    vertices = np.array([[stwo/2, twoc/2, -1.84],
+                         [sone/2, -onec/2, -1.84],
+                         [0, -1/2, -1.84],
+                         [-sone/2, -onec/2, -1.84],
+                         [-stwo/2, twoc/2, -1.84],
+                         [stwo/2, twoc/2, 1.84],
+                         [sone/2, -onec/2, 1.84],
+                         [0, -1/2, 1.84],
+                         [-sone/2, -onec/2, 1.84],
+                         [-stwo/2, twoc/2, 1.84]])
```

Comment:
instead of `1.84` on z-axes, can you use `0.5`. You might need to update the tests accordingly.